ui/src/components/mobileservices/BoundServiceRow.js

```diff
@@ -91,13 +96,76 @@ class BoundServiceRow extends Component {
     );
   }
 
+  renderBindingStatus() {
+    // check if the service is UPS.
+    // binding status is not shown for other services in the BoundServiceRow view.
+    // binding status is normally shown in UnboundServiceRow views.
+    if(!this.isUPSService()) {
+        return null;
+    }
+
+    return (
+        <ListView.InfoItem key="bind-status">
```

Comment:
Can this jsx be externalized into its own component since it is shared between bound an dunbound service rows now?

Reply:
+1